imageroot/systemd/user/nethcti-server.service

```diff
@@ -13,6 +13,9 @@ EnvironmentFile=%S/state/environment
 WorkingDirectory=%S/state
 Restart=always
 ExecStartPre=/bin/rm -f %t/nethcti-server.pid %t/nethcti-server.ctr-id
+ExecStartPre=bash -c "while [ ! -f certificates/NethServer.pem ]; do sleep 1; done"
```

Comment:
I've seen this behavior on several files, however I think that the container should handle the wait and not systemd. This can be easily achieved moving this inside the entrypoint.

Something like this should suffice:
```bash
until [ -f "/etc/certificates/<certname>" ]; do sleep 1; done
```

However, there should be some sort of timeout, something like this:
```bash
counter=0
until [ -f "/etc/certificates/<certname>" ]; do
    if [ $counter -ge <seconds_to_wait> ]; then
        echo "File '/etc/certificates/<certname>' missing."
        exit 1
    fi
    sleep 1
    counter=$((counter+1))
done
```

Reply:
I just copied existing containers to have the same behavior for all applications.
I think this is not necessary at all since the creation of certificate is done as first step inside the configure.

@Stell0 why did you add the wait loop?

Reply:
Yeah, this is not something related to this systemd in particular, due to the nature of containers I'm on the side that containers should be self-sufficient or die :skull_and_crossbones:

Reply:
Let's simplify it and remove the ExectStartPre